### PR TITLE
[ECT-99] Update events via email documentation

### PR DESCRIPTION
**Pull request by josemallma-dd**
### What does this PR do? What is the motivation?
Events via email feature was migrated to a new service, this PR updates public documentation to inform about restriction/changes applied

### Merge instructions
- [x] Merge after approval

### Additional notes
N/A

### CODE REVIEW

**Review by cmonzillo91 — APPROVED** [review]

File: content/en/service_management/events/guides/email.md [hunk 1]
```diff
@@ -75,6 +75,14 @@ Subject: Env:Test - System at 50% CPU - #test
 Body: This is a test message showing that env:test is at 50% CPU - #test
 ```
 
+### Email body processing {#email-body-2}
+The email body undergoes several cleanup steps to enhance readability and security. Expected changes include:
```
Comment (maycmlee) [inline content/en/service_management/events/guides/email.md:79]:
```suggestion
The email body goes through several cleanup steps to enhance readability and security. The expected changes include:
```

File: content/en/service_management/events/guides/email.md [hunk 2]
```diff
@@ -75,6 +75,14 @@ Subject: Env:Test - System at 50% CPU - #test
...
+
+- **HTML to Markdown:** HTML content is converted to its markdown equivalent.
+- **HTML sanitization:** For security, email bodies are sanitized, allowing only specific HTML tags: `a`, `br`, `code`, `div`, `em`, `h1`, `h2`, `h3`, `h4`, `h5`, `h6`, `hr`, `iframe`, `img`,
```
Comment (maycmlee) [inline content/en/service_management/events/guides/email.md:82]:
```suggestion
- **HTML to Markdown**: HTML content is converted to its markdown equivalent.
- **HTML sanitization**: For security, email bodies are sanitized, allowing only specific HTML tags: `a`, `br`, `code`, `div`, `em`, `h1`, `h2`, `h3`, `h4`, `h5`, `h6`, `hr`, `iframe`, `img`,
```

File: content/en/service_management/events/guides/email.md [hunk 3]
```diff
@@ -75,6 +75,14 @@ Subject: Env:Test - System at 50% CPU - #test
...
+`li`, `ol`, `p`, `pre`, `span`, `strong`, `ul`. Any other HTML tag, including strings enclosed in `<>`, are removed.
+- **Strip reply/forward content:** Only the most recent email in a thread is retained, with older replies/forwards removed.
```
Comment (maycmlee) [inline content/en/service_management/events/guides/email.md:84]:
```suggestion
- **Strip reply/forward content**: Only the most recent email in a thread is retained, with older replies and forwards removed.
```

**Review by maycmlee — COMMENTED** [review]
Left some small suggestions!

**Comment by josemallma-dd** [comment]
> Left some small suggestions!

Suggestions have been applied

**Pull request merged**